main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -741,6 +745,13 @@ DataTableView.prototype._createMenuForAllColumns = function(elmt) {
             self._collapsedColumnNames = [];
             self.render();
           }
+        },
+        {
+          label: $.i18n._('core-views')["display-null"],
+          id: "core/display-null",
+          click: function() {
+            $(".data-table-null").toggle();
+            self._shownulls = !(self._shownulls);
         }
```

Comment:
I get the following JS error here:
`Uncaught SyntaxError: Unexpected token ]`
Both with Firefox and Chromium

Reply:
Doh! I caught that error when I tested and managed not to fix it in PR. I’ll fix when I’m back at my computer. Sorry!